### PR TITLE
ARGO-182 status timeline for Endpoint groups

### PR DESCRIPTION
**Pull request by kaggis**
- Implement view controller structures for package
- Implement unittests

### CODE REVIEW

**Comment by kaggis** [comment]
please review
/cc @pkoro @makistsantekidis

File: app/statusEndpointGroups/controller.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	metricCollection := session.DB(tenantDbConfig.Db).C("status_endpoint_groups")
+
+	// Query the detailed metric results
+	err = metricCollection.Find(prepareQuery(input)).All(&results)
+	if err != nil {
+		code = http.StatusInternalServerError
+		return code, h, output, err
+	}
+
+	fmt.Println(results)
```
Comment (pkoro) [inline app/statusEndpointGroups/controller.go]: remove this line

**Comment by kaggis** [comment]
- Accept Header handler implemented
- Json output implemented 
- Licenses Added
- Unittest update and clean up

Please review
/cc @pkoro @makistsantekidis

**Comment by pkoro** [comment]
+1

**Pull request merged**